### PR TITLE
core/rawdb: db inspect add the config prefix

### PR DESCRIPTION
**Pull request by Evolution404**
Avoid the error message of `Database contains unaccounted data`
image

### CODE REVIEW

**Review by holiman — APPROVED** [review]
LGTM, thanks!

File: core/rawdb/database.go [hunk 1]
```diff
@@ -355,6 +356,8 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 			storageSnaps.Add(size)
 		case bytes.HasPrefix(key, preimagePrefix) && len(key) == (len(preimagePrefix)+common.HashLength):
 			preimages.Add(size)
+		case bytes.HasPrefix(key, configPrefix) && len(key) == (len(configPrefix)+common.HashLength):
```
Comment (rjl493456442) [inline core/rawdb/database.go:359]: Can we just count the chain config into the `meta data`? It doesn't have too much meaning to have a standalone type.

Comment (rjl493456442) [reply core/rawdb/database.go:359]: I think we also need to merge the `Shutdown metadata` into the `metadata` category. @holiman what's your opinion?

Comment (holiman) [reply core/rawdb/database.go:359]: Yes, I agree

**Comment by rjl493456442** [comment]
@Evolution404 Care to make the updates?

**Pull request closed without merging**